### DISCUSSION TITLE
3Dconnexion CadMouse support - CLOSED

### DISCUSSION THREAD

**Discussion by deluxeGitHub** [JakubAndrysek/PySpaceMouse#30] [Ideas]
Hi,
I use a Cadmouse Pro Wireless and I want to be able to control the mouse features (Smartscrolling in particular) and button remapping.

I tried to get this tool (https://github.com/mia-0/cadmousectl) to run, but with no success unfortunately.

Is it doable to make those configurations with your pyton module?

**Comment by JakubAndrysek** [comment]
I'm on vacation, I'll check it out next week. Try to remind me if I don't respond.

**Comment by JakubAndrysek** [comment]
Hi, I have looked into the [mia-0/cadmousectl](https://github.com/mia-0/cadmousectl) project, but since I do not have this mouse, I am not planning to implement this mouse into the project. But feel free to implement it as PR.

Have you tested this updated forked version? → https://github.com/guillaumechauvat/cadmousectl

**Comment by peter-mitrano-ar** [comment]
To me it looks like the CadMouse is a normal 2D mouse -- it would not make sense to integrate it into this repository, which is focused solely on the 3D HID input devices.
